src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -534,6 +581,57 @@ export default class ComparisonGroupManager extends React.Component<
         this.props.store.deleteMarkedComparisonGroups();
     }
 
+    @computed get anyThereAnySharedGroups() {
```

Comment:
usually I've been phrasing this like `existSharedGroups` but I'm not sure that there's an established idiom for this. maybe let's go with `existSharedGroups` to start a trend?